test-suite/src/function/concat_ws.rs

```diff
@@ -0,0 +1,92 @@
+use {
+    crate::*,
+    gluesql_core::{prelude::Value::*, translate::TranslateError},
+};
+
+test_case!(concat_ws, async move {
+    test!(
+        r#"VALUES(CONCAT_WS(",", "AB", "CD", "EF"))"#,
+        Ok(select!(
+            column1
+            Str;
+            "AB,CD,EF".to_owned()
+        ))
+    );
+
+    run!(
+        "
+        CREATE TABLE Concat (
+            id INTEGER,
+            rate FLOAT,
+            flag BOOLEAN,
+            text TEXT,
+            null_value TEXT NULL,
+        );
+    "
+    );
+    run!(r#"INSERT INTO Concat VALUES (1, 2.3, TRUE, "Foo", NULL);"#);
```

Comment:
I missed in the previous review. Everything else looks nice.

It looks that all the current tests are only using literals.
However, you've already added values like this so could we have tests which use these `id`, `rate`, `flag`, `text` or `null_value` field?